src/components/LicensesFieldArray/AmendmentsFieldArray.js

```diff
@@ -78,18 +73,18 @@ class AmendmentsFieldArray extends React.Component {
       license = {},
       name
     } = this.props;
+    const { statusWarnings } = this.state;
     const { amendments = [] } = license;
-
-    if (!items.length) return null;
-
+    if (!items.length) {
```

Comment:
Do you have an automatic code formatter enabled or something? Might want to leave it off since it ended up making this section a bit less readable.

Reply:
Yep, this makes sense. I wasn't aware that we were certain that the order would never change, hence why I went to the extra effort. This is a lot cleaner though. I wasn't aware that we could set, say, Array[2] = "whatever" on a potentially empty array... presumably there's some magic happening in the background there to make the Array [undefined, undefined, "whatever"]...? Also, for some reason the implicit setting of `warnings` in `this.setState` didn't work, so I changed it to be `return {warnings: warnings}`.